packages/web/components/Player/Filter/PlayerFilter.tsx

```diff
@@ -122,86 +125,273 @@ export const PlayerFilter: React.FC<Props> = ({
+                    } else {
+                      setQueryVariable('search', `%%`);
+                    }
+                  }}
+                  placeholder="SEARCH NAME OR ETH ADDRESS"
+                  _placeholder={{ color: 'whiteAlpha.500' }}
+                  size="md"
+                  type="text"
+                  value={search}
+                  w="100%"
                 />
-              </InputRightElement>
-            )}
-          </InputGroup>
-          <MetaButton
-            type="submit"
-            size="lg"
-            isDisabled={fetching}
-            px="16"
-            display={isSmallScreen ? 'none' : 'flex'}
+                {search.length > 0 && (
+                  <InputRightElement>
+                    <IconButton
+                      p="2"
+                      variant="link"
+                      colorScheme="white"
+                      icon={<CloseIcon />}
+                      onClick={() => {
+                        setSearch('');
+                        setQueryVariable('search', '%%');
+                      }}
+                      aria-label="Clear Search"
+                    />
+                  </InputRightElement>
+                )}
+              </InputGroup>
+            </Form>
+          </Box>
+          <Box
+            flexGrow={{
+              '2xl': '2', // take up more space when side-by-side with search on wide screens
+            }}
           >
-            SEARCH
-          </MetaButton>
-        </Stack>
-      </Form>
-      <DesktopFilters
-        display={isSmallScreen ? 'none' : 'flex'}
-        {...{
-          aggregates,
-          skills,
-          setSkills,
-          playerTypes,
-          setPlayerTypes,
-          timeZones,
-          setTimeZones,
-          availability,
-          setAvailability,
-          sortOption,
-          setSortOption,
-        }}
-      />
+            {/**
+             * Put in the filters button or the DesktopFilters, depending on the screen size
+             * The button has a label 'Filters' if the screen is wide enough to fit it
+             * The FiltersIcon has a marginTop to align it in the middle
+             */}
+            {isSmallScreen ? (
+              // Filters button to open the menu on small screens
+              <Button
+                variant="outline"
+                borderColor="blueLight"
+                borderRadius={4}
+                borderWidth="1px"
+                color="white"
+                onClick={onOpen}
+                fontSize="sm"
+                fontWeight="400"
+                px={3}
+                minH="2.5rem"
+                size="md"
+              >
+                <FiltersIcon marginTop="1px" />
+                <Text
+                  as="span"
+                  display={{
+                    base: 'none',
+                    sm: 'inline',
+                  }}
```

Comment:
```suggestion
                  display={['none', 'inline']}
```

Reply:
I changed this, but I do prefer having the breakpoint names written into it. It's clearer for me.